release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -2941,8 +2975,10 @@ It is anticipated that an upcoming z-stream release will include a fix for this
 * There is a known issue after enabling and disabling IPsec during runtime that causes the cluster to be in an unhealthy state with the error message: `an unknown error has occurred: MultipleErrors`. (link:https://issues.redhat.com/browse/OCPBUGS-19408[*OCPBUGS-19408*])
 
 * Creating pods with Microsoft Azure File NFS volumes that are scheduled to the control plane node causes the mount to be denied.
+
+* For clusters that run on {rh-openstack} 17.1 and use network function virtualization (NFV), a known issue in {rh-openstack} prevents succesful cluster deployment. There is no workaround for this issue. Contact Red Hat Support to request a hotfix. Additionally, there is no support for Kuryr installations on {rh-openstack} 17.1.
 +
-To work around this issue: If your control plane nodes are schedulable, and the pods can run on worker nodes, use `nodeSelector` or Affinity to schedule the pod in worker nodes.(link:https://issues.redhat.com/browse/OCPBUGS-18581[*OCPBUGS-18581*])
+To work around this issue: If your control plane nodes are schedulable, and the pods can run on worker nodes, use `nodeSelector` or Affinity to schedule the pod in worker nodes. (link:https://issues.redhat.com/browse/OCPBUGS-18581[*OCPBUGS-18581*])
```

Comment:
Looks like this workaround refers to the previous issue "Creating pods with Microsoft Azure File NFS volumes that are scheduled to the control plane node causes the mount to be denied.". In that case it should be placed just after the previous issue.

Reply:
Good catch. We spend just about every minute of the week up to freeze resolving merge conflicts--must be a mistake in that.